tools/install.sh

```diff
@@ -328,8 +328,8 @@ done;
 
 
 echo ""
-echo "Attempting to create and grant privileges to MySQL user '$mysqluser'@'localhost' ..."
-echo "GRANT UPDATE,INSERT,SELECT,DELETE ON $mysqldb.* TO '$mysqluser'@'localhost' IDENTIFIED BY '$mysqlpass' WITH GRANT OPTION" | mysql $mysqldb -h$mysqlhost --user=$mysqlrootuser --password="$mysqlrootpass" -A > /dev/null 2>&1
+echo "Attempting to create and grant privileges to MySQL user '$mysqluser'@'$mysqlhost' ..."
+echo "GRANT UPDATE,INSERT,SELECT,DELETE ON $mysqldb.* TO '$mysqluser'@'$mysqlhost' IDENTIFIED BY '$mysqlpass' WITH GRANT OPTION" | mysql $mysqldb -h$mysqlhost --user=$mysqlrootuser --password="$mysqlrootpass" -A > /dev/null 2>&1
```

Comment:
This looks like it's supposed to be a different pull request, can you separate it and take it out of this one?

(Also, I don't think this change will work.. the @ part is the host that the user is connecting from, not the MySQL host.)